### PR TITLE
feat(plasma-tokens): Add sbermarket tenants

### PR DESCRIPTION
**Pull request by neretin-trike**
Создание дополнительных трёх тем для сбермаркета и удаление префиксов `plasma-colors` у css переменных для тем, который были созданы с помощью генератора и лежат в директории `plasma-tokens/data/themes`

:baby_chick: Download canary assets:

[colors--canary.217.3647001361.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.217.3647001361.xml)  
[color_metro_ios-swift--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.217.3647001361.swift)  
[color_metro_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.217.3647001361.kt)  
[color_metro_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.217.3647001361.ts)  
[color_sbermarket_ios-swift--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.217.3647001361.swift)  
[color_sbermarket_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.217.3647001361.kt)  
[color_sbermarket_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.217.3647001361.ts)  
[color_sberprime_ios-swift--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.217.3647001361.swift)  
[color_sberprime_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.217.3647001361.kt)  
[color_sberprime_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.217.3647001361.ts)  
[color_selgros_ios-swift--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.217.3647001361.swift)  
[color_selgros_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.217.3647001361.kt)  
[color_selgros_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.217.3647001361.ts)  
[color_smbusiness_ios-swift--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.217.3647001361.swift)  
[color_smbusiness_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.217.3647001361.kt)  
[color_smbusiness_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.217.3647001361.ts)  
[PlasmaTokensColor--canary.217.3647001361.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.217.3647001361.swift)  
[typo_mage_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.217.3647001361.kt)  
[typo_mage_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.217.3647001361.ts)  
[typo_plasma_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.217.3647001361.kt)  
[typo_plasma_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.217.3647001361.ts)  
[typo_ruler_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.217.3647001361.kt)  
[typo_ruler_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.217.3647001361.ts)  
[typo_sage_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.217.3647001361.kt)  
[typo_sage_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.217.3647001361.ts)  
[typo_sbermarket_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.217.3647001361.kt)  
[typo_sbermarket_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.217.3647001361.ts)  
[typo_soulmate_kotlin--canary.217.3647001361.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.217.3647001361.kt)  
[typo_soulmate_react-native--canary.217.3647001361.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.217.3647001361.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.117.0-canary.217.3647001361.0
  npm install @salutejs/plasma-core@1.84.0-canary.217.3647001361.0
  npm install @salutejs/plasma-icons@1.111.0-canary.217.3647001361.0
  npm install @salutejs/plasma-temple@1.116.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-b2b@1.5.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-b2c@0.14.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-web@1.20.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens@1.29.0-canary.217.3647001361.0
  npm install @salutejs/plasma-ui@1.149.0-canary.217.3647001361.0
  npm install @salutejs/plasma-web@1.145.0-canary.217.3647001361.0
  npm install @salutejs/plasma-cy-utils@0.35.0-canary.217.3647001361.0
  npm install @salutejs/plasma-sb-utils@0.82.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-android@2.36.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-ios-swift@2.36.0-canary.217.3647001361.0
  npm install @salutejs/plasma-tokens-utils@0.14.0-canary.217.3647001361.0
  # or 
  yarn add @salutejs/plasma-b2c@1.117.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-core@1.84.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-icons@1.111.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-temple@1.116.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-b2b@1.5.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-b2c@0.14.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-web@1.20.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens@1.29.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-ui@1.149.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-web@1.145.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-cy-utils@0.35.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-sb-utils@0.82.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-android@2.36.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.36.0-canary.217.3647001361.0
  yarn add @salutejs/plasma-tokens-utils@0.14.0-canary.217.3647001361.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-217/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-217/
  ui storybook: http://plasma.sberdevices.ru/pr-217/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-217/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-217/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-217/temple-storybook/

File: utils/plasma-tokens-utils/src/generators/generateColorThemes.ts [hunk 1]
```diff
@@ -14,14 +14,17 @@ export const generateColorThemes = (colorThemes: Record<string, TokenDataGroup<s
     let indexContent = ROBO_COMMENT;
 
     for (const [fileName, theme] of Object.entries(colorThemes)) {
+        const fromData = Boolean(theme.fromData);
+        delete theme.fromData;
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/generators/generateColorThemes.ts:18]:
```
const { _fromData, ...them } = _theme;
const fromData = Boolean(_fromData);
```
но я не настаиваю, просто не люблю `delete`

**Review by Yeti-or — APPROVED** [review]
только нужно спросить норм ли ребятам из Маркета

File: utils/plasma-tokens-utils/src/generators/generateColorThemes.ts (hunk 1 above)
Comment (Yakutoc) [reply utils/plasma-tokens-utils/src/generators/generateColorThemes.ts:18]: Поддерживаю

**Comment by Yakutoc** [comment]
@neretin-trike Напиши пожалуйста коротко для чего и что делает PR&

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-217/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-217/
  ui storybook: http://plasma.sberdevices.ru/pr-217/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-217/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-217/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-217/temple-storybook/

File: utils/plasma-tokens-utils/src/generators/generateColorThemeValues.ts [hunk 2]
```diff
@@ -9,7 +9,9 @@ import { generateTokens } from '../generation';
 export const generateColorThemeValues = (colorThemes: Record<string, TokenDataGroup<string>>) => {
     const out: GeneratedFiles = [];
 
-    for (const [name, theme] of Object.entries(colorThemes)) {
+    for (const [name, themeItem] of Object.entries(colorThemes)) {
+        const { fromData, ...theme } = themeItem;
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/generators/generateColorThemeValues.ts:13]:
```
const { fromData: _ , ...theme} = themeItem;
```

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-217/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-217/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-217/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-217/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-217/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-217/temple-storybook/

**Pull request merged**